### PR TITLE
Reconcile JSON functions doc

### PR DESCRIPTION
**Pull request by diana-macrometa**
Continuing to reconcile Functions and Extensions.

The JSON mapper extension was present as a Sink Mapper and Source Mapper in the Functions folder.

- The only thing missing in the Functions docs were some text from the examples. I added that and fixed some formatting.
- Deleted the redundant json.md doc.

### CODE REVIEW

**Comment by diana-macrometa** [comment]
@StoyanTc or @ashish-macrometa, will you be able to review this soon?

File: docs/cep/query-guide/functions/sinkmapper/json.md [hunk 1]
```diff
@@ -18,9 +19,41 @@ Syntax
 
 ## Example 1
 
-    CREATE SINK FooStream WITH (type='inMemory', topic='stock', map.type='json') (symbol string, price float, volume long);
+```js
+CREATE SINK FooStream WITH (type='inMemory', topic='stock', map.type='json') (symbol string, price float, volume long);
```
Comment (StoyanTc) [inline docs/cep/query-guide/functions/sinkmapper/json.md]: Let's replace `inMemory` by `stream`.

File: docs/cep/query-guide/functions/sinkmapper/json.md [hunk 2]
```diff
@@ -18,9 +19,41 @@ Syntax
...
+    "price":55.6,        
+    "volume":100    
+    }
+}
+```
 
 ## Example 2
 
+```js
     CREATE SINK BarStream WITH (type='inMemory', topic='{{symbol}}', map.type='json', map.enclosing.element='$.portfolio', map.validate.json='true', map.payload="""{"StockData":{"Symbol":"{{symbol}}","Price":{{price}}}}""") (symbol string, price float, volume long);
```
Comment (StoyanTc) [inline docs/cep/query-guide/functions/sinkmapper/json.md]: Let's replace `inMemory` by `stream`.

File: docs/cep/query-guide/functions/sourcemapper/json.md [hunk 3]
```diff
@@ -23,43 +27,116 @@ Syntax
 
 ## Example 1
 
+```js
     CREATE SOURCE FooStream WITH (type='inMemory', topic='stock', map.type='json') (symbol string, price float, volume long);
```
Comment (StoyanTc) [inline docs/cep/query-guide/functions/sourcemapper/json.md]: Let's replace `inMemory` by `stream`.

File: docs/cep/query-guide/functions/sourcemapper/json.md [hunk 4]
```diff
@@ -23,43 +27,116 @@ Syntax
...
+        "symbol":"GDN",    
+        "price":55.6,    
+        "volume":100
+    }
+```
 
 ## Example 2
 
+```js
     CREATE SOURCE FooStream WITH (type='inMemory', topic='stock', map.type='json') (symbol string, price float, volume long);
```
Comment (StoyanTc) [inline docs/cep/query-guide/functions/sourcemapper/json.md]: Let's replace `inMemory` by `stream`.

File: docs/cep/query-guide/functions/sourcemapper/json.md [hunk 5]
```diff
@@ -23,43 +27,116 @@ Syntax
...
+        {"symbol":"GDN","price":55.6,"volume":100},
+        {"symbol":"GDN","price":56.6,"volume":99},
+        {"symbol":"GDN","price":57.6,"volume":80}
+    ]
+```
 
 ## Example 3
 
+```js
     CREATE SOURCE FooStream WITH (type='inMemory', topic='stock', map.type='json', map.enclosing.element="$.portfolio", map.attributes="symbol = 'company.symbol', price = 'price', volume = 'volume'")
```
Comment (StoyanTc) [inline docs/cep/query-guide/functions/sourcemapper/json.md]: Please replace by 
```
CREATE SOURCE FooStream WITH (type='stream', topic='stock', map.type='json', enclosing.element="$.portfolio", map.attributes.symbol="company.symbol", map.attributes.price="price", map.attributes.volume="volume" ) (symbol string, price float, volume long);
```

File: docs/cep/query-guide/functions/sourcemapper/json.md [hunk 6]
```diff
@@ -23,43 +27,116 @@ Syntax
...
+                "price":55.6       
+            }   
+        }
+    }
+```
 
 ## Example 4
 
+```js
     CREATE SOURCE FooStream WITH (type='inMemory', topic='stock', map.type='json', map.enclosing.element="$.portfolio", map.attributes="symbol = 'stock.company.symbol', price = 'stock.price', volume = 'stock.volume'") (symbol string, price float, volume long);
```
Comment (StoyanTc) [inline docs/cep/query-guide/functions/sourcemapper/json.md]: Please replace the example by 
```
CREATE SOURCE FooStream WITH (type='inMemory', topic='stock', map.type='json', enclosing.element="$.portfolio", map.attributes.symbol = 'stock.company.symbol', map.attributes.price = 'stock.price', map.attributes.volume = 'stock.volume') (symbol string, price float, volume long);```

**Review by StoyanTc — APPROVED** [review]
@diana-macrometa - We must not provide `inMemory` type into the examples but `stream`. One could crash the `CEP` using `inMemory` exhausting all available memory.

**Pull request merged**